cosmosdb/Module.md

```diff
@@ -54,7 +54,7 @@ designed to store and query JSON-like documents. Therefore, the document you cre
     int valueOfPartitionKey = 0;
     string id = "my_document";
 
-    check azureCosmosClient-> createDocument("my_database", "my_container", id, document, valueOfPartitionKey);
+    cosmosdb:DocumentResponse response = check azureCosmosClient-> createDocument("my_database", "my_container", id, document, valueOfPartitionKey);
     ```
 **Note:** <br/>
 - This document is created inside an already existing container with ID **my_container** and the container was created inside a database with ID **my_document**.
```

Comment:
Update the database ID to "my_database". This was an error from the beginning. Users who are new to cosmos DB sometimes gets confused by it :)

Reply:
Updated in my_database